dev_guide/managing_images.adoc

```diff
@@ -968,24 +967,9 @@ ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicate
 An image stream can be configured to import tag and image metadata from private
 image registries, requiring authentication.
 
-To configure this, you need to create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret]
-which is used to store your credentials.
-
-Create the secret first, before importing the image from the private repository:
-
-----
-$ oc create secret docker-registry <secret_name> \
-    --docker-server=<docker_registry_server> \
-    --docker-username=<docker_user> \
-    --docker-password=<docker_password> \
-    --docker-email=<docker_email>
-----
-
-For more options, see:
-
-----
-$ oc create secret docker-registry --help
-----
+To configure this, you need to create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret],
+which is used to store your credentials. See xref:#allowing-pods-to-reference-images-from-other-secured-registries[here]
```

Comment:
Thanks, @soltysh! Would you mind changing this sentence to the following or something similar? Otherwise, LGTM.  

"See xref:#allowing-pods-to-reference-images-from-other-secured-registries[Allowing Pods to Reference Images from Other Secured Registries] for instructions on creating a secret using `oc create secret` command."